### PR TITLE
Starting integration testing comment controller layer.

### LINKED ISSUE #30: [Testing] Comment Controller Test
### Resources: 
Resource from Azhya: 
[Spring Boot Testing Basics: How to Unit Test & Integration Test REST Controllers](https://www.youtube.com/watch?v=pNiRNRgi5Ws)

Additional Resources:
[Test Driven Development with Spring Boot - Sannidhi Jalukar, Madhura Bhave](https://www.youtube.com/watch?v=s9vt6UJiHg4)

-----
### Methods: 
As a small reminder, we only need the minimum amount of controller code needed to actually run the test (Calvin's example should show that). 
Note: It's expected to fail. 
Action item: Please go ahead and set up the minimum amount of code (in the Controller layer) for your assigned testing table that you will need to create your test. 

Your tests should have the following methods: 
- getAll
- getById
- Create
- Update
- Delete

### PR DESCRIPTION
**Pull request by L3viathon**

### CODE REVIEW

**Review by pyaeger — APPROVED** [review]
Thank you.

File: src/test/java/com/amplifier/controllers/CommentControllerIntTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        mockDb.add(mockComment1);
+        mockDb.add(mockComment2);
+   }
+   @Test
+   @Order(1)
+   @DisplayName("1. AppContext")
+   public void contextLoads() {
+           assertThat(commentController).isNotNull();
+}
+
```
Comment (FullSol) [inline src/test/java/com/amplifier/controllers/CommentControllerIntTest.java:232]: Good so far.

**Review by FullSol — CHANGES_REQUESTED** [review]

File: src/test/java/com/amplifier/controllers/CommentControllerIntTest.java (hunk 1 above)
Comment (FullSol) [reply src/test/java/com/amplifier/controllers/CommentControllerIntTest.java:232]: Pending completion.

**Comment by L3viathon** [comment]
More tests needed to be added with Julian's additional methods.

File: src/test/java/com/amplifier/controllers/CommentControllerIntTest.java [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    @Test
+    @Order(7)
+    @DisplayName("7. Attempt to update comment - failed")
+    public void updateComment_Failed() throws Exception {
+            //CommentService method needs to be merged in.
+            when(commentService.update(mockCommentUpdation)).thenReturn(true);
+
+            RequestBuilder request = MockMvcRequestBuilders
+                            .put("/api/v1/comment?id=3")
```
Comment (FullSol) [inline src/test/java/com/amplifier/controllers/CommentControllerIntTest.java:171]: Is patch available here?

File: src/test/java/com/amplifier/controllers/CommentControllerIntTest.java [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+    @Test
+    @Order(8)
+    @DisplayName("8. Attempt to update comment - passed")
+    public void updateComment_Passed() throws Exception {
+            //CommentService method needs to be merged in.
+            when(commentService.update(mockCommentUpdation)).thenReturn(true);
+
+            RequestBuilder request = MockMvcRequestBuilders
+                            .put("/api/v1/comment?id=3")
```
Comment (FullSol) [inline src/test/java/com/amplifier/controllers/CommentControllerIntTest.java:189]: Same here

**Review by FullSol — APPROVED** [review]

**Comment by FullSol** [comment]
@L3viathon i left a comment, but approving

**Pull request merged**